addons/l10n_it_edi_sdicoop/models/res_config_settings.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+
+from odoo import api, models, fields
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    is_edi_proxy_active = fields.Boolean(compute='_compute_is_edi_proxy_active')
+
+    @api.depends('company_id.account_edi_proxy_client_ids', 'company_id.account_edi_proxy_client_ids.active')
+    def _compute_is_edi_proxy_active(self):
+        for config in self:
+            config.is_edi_proxy_active = config.company_id.account_edi_proxy_client_ids
```

Comment:
Should not we filter on the right edi format here?  We also need to be sure that if the user is inactive, we can make a new contract.

Reply:
No need to filter right now, because it's only supported for Italy. To do it for multiple formats, it has some functional implication (as to when we should create the proxy user, etc), and we will probably need to change the way this works, so I tried to kept it simple for now.

Reply:
And if the user is inactive, making a new contract is supported.